packages/libs/react-ui/src/components/Grid/GridItem.tsx

```diff
@@ -23,17 +24,20 @@ const assembleColumnSpanVariants = (
 };
 
 export const GridItem: FC<IGridItemProps> = ({
+  className,
   children,
   columnSpan,
   rowSpan = 1,
 }) => {
-  const className = classNames(
-    gridItemClass,
-    rowSpanVariants[rowSpan],
-    columnSpan && assembleColumnSpanVariants(columnSpan),
-  );
   return (
-    <div className={className} data-testid="kda-grid-item">
```

Comment:
The reason is because we don't want to clutter the html with unnecessary attributes and this encourages us to test based on aria roles which aligns more with actual user interaction and helps us put more focus on a11y
There are some of the threads where this is discussed on slack:
https://kadena-io.slack.com/archives/C05DZR7GPFH/p1691567264887629
https://kadena-io.slack.com/archives/C05DZR7GPFH/p1696588154162339

Reply:
Thanks for the clarification 🙌